lib/fastly_nsq/message_queue/strategy.rb

```diff
@@ -1,12 +1,34 @@
-class Strategy
-  def self.for_queue
-    if MessageQueue::FALSY_VALUES.include?(ENV['FAKE_QUEUE'])
-      Nsq
-    elsif MessageQueue::TRUTHY_VALUES.include?(ENV['FAKE_QUEUE'])
-      FakeMessageQueue
-    else
-      message = "You must set ENV['FAKE_QUEUE'] to either true or false"
-      raise InvalidParameterError, message
-    end
+module MessageQueue::Strategy
+  module_function
+
+  def for_queue
+    real_queue || fake_queue || error
+  end
+
+  private_class_method
```

Comment:
[Based on the source][1], this looks like it takes a symbol as an argument, like

```ruby
class Foo
  private_class_method(:method_name)
  # ...
```

Is there a missing argument or am I misreading the usage?

[1]: http://ruby-doc.org/core-2.3.1/Module.html#method-i-private_class_method